agb/src/display/tiled/vram_manager.rs

```diff
@@ -419,7 +419,7 @@ impl VRamManager {
         }
     }
 
-    fn set_background_palette(&mut self, pal_index: u8, palette: &palette16::Palette16) {
+    pub fn set_background_palette(&mut self, pal_index: u8, palette: &palette16::Palette16) {
```

Comment:
I've just realised that this method, as-is is unsafe. Would you mind also adding an `assert!(pal_index < 16)` in here?

Reply:
Done.